draft-beky-httpbis-metadata.md

```diff
@@ -31,7 +31,10 @@ author:
 --- abstract
 
 This document describes a mechanism to send meta information over HTTP/2
-({{!RFC9113}}) and HTTP/3 ({{!RFC9114}}) connections.  TODO elaborate
+({{!RFC9113}}) and HTTP/3 ({{!RFC9114}}) connections that refers either to the
+entire connection or a specific stream without changing the semantics of the
```

Comment:
uber-nit: "entire connection or to a specific" ("to" for grammatical parallelism).

maybe better: "refers to either the entire connection or a specific stream" (also has parallelism).

Reply:
Oh nice, I always liked parallelism, and now I learned its name!